borg/helpers.py

```diff
@@ -669,7 +671,10 @@ def decorated_function(*args):
 @memoize
 def uid2user(uid, default=None):
     try:
-        return pwd.getpwuid(uid).pw_name
+        if sys.platform != 'win32':
+            return pwd.getpwuid(uid).pw_name
+        else:
+            return os.getlogin()
```

Comment:
maybe we could use something obviously wrong, instead of something maybe and non-obviously wrong. like "dummy"  # TODO